gallery/setupTests.js

```diff
@@ -12,7 +12,10 @@ jest.retryTimes(3);
 
 const toMatchImageSnapshot = configureToMatchImageSnapshot({
   customDiffDir: path.join(__dirname, 'test-results'),
-  customSnapshotIdentifier: ({ defaultIdentifier }) => defaultIdentifier
+  // eslint-disable-next-line no-undef
+  customSnapshotsDir: process.env.THEME === 'DARK' && path.join(__dirname, 'visualtests', '__image_snapshots_dark__'),
+  // eslint-disable-next-line no-undef
+  customSnapshotIdentifier: ({ defaultIdentifier }) => defaultIdentifier + (process.env.THEME === 'DARK' ? '-dark' : '')
```

Comment:
Do we need both of these, the custom directly and custom snapshot naming?

Reply:
> Do we need both of these, the custom directory and custom snapshot naming?

I thought it may be a good idea that every snapshot have a unique name to avoid any confusion searching for snapshots etc, but I can remove the -dark suffix if you prefer